.github/DEVELOPMENT.md

```diff
@@ -74,70 +74,89 @@ Your command line application will then display some information about Eleventy,
 
 You can also use the `External` URL to preview the local copy on another device connected to the same network, which helps you check to see how the site looks and functions on things like smartphones. This is done via [Browsersync](https://www.browsersync.io/).
 
-### Different npm scripts
+### Updating the website
 
-This is a list of all the **npm** commands you could run from the command line within this project.
+Saving project files will make Node.js regenerate the website to reflect the changes you made. Your command line application will display some new messaging to reflect this, including any errors you might accidentally make. Don't worry! Since the site uses version control, you're not in danger of seriously breaking anything. If you fix the error, Node.js should continue to run.
 
-#### Building the site
+Make sure you edit the files in the `src/` subdirectory. Any edits made in the `_site` subdirectory will be overwritten by the next change to any file in `src/` and all your hard work will be lost!
 
-The `npm run build` command will build the site into the `_site` folder. It is used for the production server hosted with Github pages.
+### Quitting
 
-#### Running in development mode
+You can tell Node.js to stop running by pressing the <kbd>Control</kbd> and <kbd>C</kbd> keys at the same time in your command line application, or by closing the command line application window or tab.
 
-The `npm run dev` command will build the site into the `_site` folder and set the `ELEVENTY_ENV=dev` variable for the Netlify staging instance of the site.
+## Alerts
 
-#### Building the HTML files
+We have three types of alerts that can be used either individually or together in your webpage, as per your needs and their logic can be found in [alert include](https://github.com/gc-da11yn/gc-da11yn.github.io/blob/main/src/_includes/partials/alerts.njk)
 
-The `npm run eleventy` command builds the HTML pages from the `src` folder into the `_site`.
+### 1. isDraft
+#### What is the function of isDraft
 
-#### Creating a new folder and the files needed
+The isDraft variable is used to control whether or not an alert message includes a isDraft message. If the isDraft variable is set to true, the alert message will include the message that the content is still in draft and is not final. If it is  included not at all, the "Draft" message will be excluded
```

Comment:
Can we add the ` around the all the instances of the front-matter values in this document?

Add period at the end of this paragraph.

```suggestion
The `isDraft` variable is used to control whether or not an alert message includes a `isDraft` message. If the `isDraft` variable is set to `true`, the alert message will include the message that the content is still in draft and is not final. If it is  included not at all, the "Draft" message will be excluded.
```